### ISSUE TITLE
Callback firing multiple times for the same message when using Wit.ai

### ISSUE THREAD

**Issue by lmmfranco** [Rei-x/discord-speech-recognition#17]
DiscordSR seems to be sending multiple requests when using wit.ai.

Code used for this test:
```js
const { Client, Intents } = require('discord.js');
const { addSpeechEvent, resolveSpeechWithWitai } = require("discord-speech-recognition");
const { joinVoiceChannel } = require("@discordjs/voice");

const client = new Client({
    intents: [
        Intents.FLAGS.GUILDS,
        Intents.FLAGS.GUILD_VOICE_STATES,
        Intents.FLAGS.GUILD_MESSAGES,
    ],
});

addSpeechEvent(client, {
    speechRecognition: resolveSpeechWithWitai,
    lang: 'pt-BR',
    key: '<my-witai-key>',
    profanityFilter: false,
});


client.on('messageCreate', (msg) => {
    console.log("[M]", msg.member.displayName, msg.content);
    if (msg.content.startsWith('##join') && msg.member?.voice.channel) {
        joinVoiceChannel({
            channelId: msg.member.voice.channelId,
            guildId: msg.member.guild.id,
            adapterCreator: msg.member.guild.voiceAdapterCreator,
            selfDeaf: false,
            selfMute: true,
        });
    }
});

client.on('speech', (msg) => {
    console.log(`[S] ${msg.member.displayName}: ${msg.content}`);
});

client.on("ready", () => {
    console.log("BOT READY");
})

client.login('<my-bot-key>');
```

Resulting log:
```
BOT READY
[M] darkbreaker ##join
[S] darkbreaker: oi teste um dois três
[S] darkbreaker: oi teste um, dois, três
```

Both even show up on wit.ai dashboard:  
![image](https://user-images.githubusercontent.com/5719409/136720406-088a3b0b-12fa-42a6-a96b-ff3dd5c1a50c.png)

When using just ```addSpeechEvent(client);``` with the default google SR it worked as intended:
```
BOT READY
[M] darkbreaker ##join
[S] darkbreaker: this is a test
```

**Comment by Rei-x** [comment]
I'm sorry, but I can't reproduce it. When I run your code, I get correct output:
![image](https://user-images.githubusercontent.com/38581479/136810360-515f6f4b-e280-4d4d-9625-2a80f2a4d413.png)
Do you use the latest version of the package?

And also, you don't have to specify language for wit.ai in speechOptions (cuz it doesn't have any effect), you do it in your app settings:
![image](https://user-images.githubusercontent.com/38581479/136811347-607baf95-efe2-42aa-b069-31726eb08565.png)

**Comment by Rei-x** [comment]
Fixed in #18
